README.md

```diff
@@ -1 +1,144 @@
+
+# 📖 Leader board App <a name="leaderboard-app"></a>
+**Leader board App**  A JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. Developed a first working version of the app following a wireframe, but with minimal styling - just focus on functionality. Consumes the Leaderboard API using JavaScript async and await.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>Javascript</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Has a page with a list of scores**
+- **Has a section where a user can add a new score**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-[Live Demo Link](https://veronica365.github.io/leaderboard-app/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A [browser](https://www.google.com/search?q=what+is+a+browser&oq=what+is+a+browser&aqs=chrome..69i57.2748j0j1&sourceid=chrome&ie=UTF-8) of your choice
+- That you have set up Git on you desired computer
+- [NODEJS 16.10](https://nodejs.org/en) or higher
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-desired-folder
+  git clone git@github.com:veronica365/leaderboard-app.git
+```
+
+### Install
+
+Install this project dependencies with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+- open the repo folder `leaderboard-app`
+- open the dist folder
+- Rightclick on the index.html file and select open in the browser
+
+### Run tests
+
+To run tests to check the lint errors, run the following command:
+
+```sh
+  npx eslint . --fix
+```
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+  GitHub Pages
+
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [William](https://github.com/verocnica365)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Adding test to the application
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/veronica365/leaderboard-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- Please provide a valid MIT license link to your README; the current link is broken. 👍🏽 
![Screenshot from 2023-03-08 17-39-38](https://user-images.githubusercontent.com/44624138/223774773-25fb9bd8-5d4f-4407-b058-1f2f3321c3eb.png)